ceph.spec.in

```diff
@@ -1238,6 +1238,23 @@ fi
 %postun -n librados2
 /sbin/ldconfig
 
+%files -n ceph-plugins-devel
```

Comment:
Minor nit: there should be a line of hash symbols above this section, as with other %files sections